packages/core/src/components/card/card.md

```diff
@@ -16,7 +16,7 @@ element.
 
 Add the `.@ns-interactive` modifier class to make a `.@ns-card` respond to user interactions. When you
 hover over cards with this class applied, the mouse changes to a pointer and the elevation shadow on
-the card increases by two levels.
+the card increases to the maximum level.
```

Comment:
this isn't technically accurate, so I kinda think we should go even more minimal:

> the elevation shadow on the card increases.

But it's much better than current so approving anyway

Reply:
how is it not correct? it always goes to 3, the maximum.